semgrep/semgrep/cli.py

```diff
@@ -64,23 +64,29 @@ def cli() -> None:
     )
 
     parser.add_argument(
-        "--include",
+        "--exclude",
```

Comment:
These changes are just me standardizing the order to exclude, include, exclude-dir, include-dir.